tools/common-tools/src/demos/index.js

```diff
@@ -1,11 +1,13 @@
 import LogsViewer from "./logsviewer";
 import lldSignature from "./lld-signature";
-import repl from "./repl";
+//import repl from "./repl";
 import ethtxtools from "./eth-tx-tools";
+import sync from "./sync";
 
 export default {
   LogsViewer,
   lldSignature,
-  repl,
+  //repl,
```

Comment:
this makes absolutely no sense but if i don't comment repl after adding the sync folder. I now get this:

```

./src/demos/repl/live-common-setup.js
Cannot find module: '@ledgerhq/hw-transport-webusb'. Make sure this package is installed.

You can install this package by running: npm install @ledgerhq/hw-transport-webusb.

```

maybe we have some sort of circular dep issue? or is it because of webpack somehow? but it worked BEFORE adding the sync/ demo (which depends more strongly on live-common then 🤔  )

maybe you know @jelbaz-ledger ?